### PR TITLE
Add font awesome and display options

### LINKED ISSUE #77: add Font awesome icons in sensor name
**Is your feature request related to a problem? Please describe.**
In some case, we don't want to show the value, but instead an icon and a mapping, For example, with a light, or a lock. You don't need to show the on/off value, juste just a yellow icon

**Describe the solution you'd like**
In the previous version, I used a lot of awesome-icon with no value and more than one mapping, this was a very usefull feature.

**Describe alternatives you've considered**
Some alternative could be multiple mapping with image rendering and an optional value display.

**Additional context**

### LINKED ISSUE #80: No possibility any more to not render the value of a sensor in version 1.0.1
In previous versions it was possible to suppress the display of a sensor's value, i.e. just showing the background. This is no more possible in version 1.0.1
Could you please add this feature again?

image

### PR DESCRIPTION
**Pull request by zfleet**
- Add font awesome library
- Add iconName prop to sensor
- Add displayValue prop to sensor
- Add displayName prop to sensor
- Fix custom editor with new props
Resolves #77 
Resolves #80

### CODE REVIEW

**Comment by zfleet** [comment]
I updated dependencies to 7.3, that's why CI fail

**Comment by zfleet** [comment]
Do you think you'll merge this PR and publish a new version ?

File: src/customEditors/EditorSensorItem.tsx [hunk 1]
```diff
@@ -65,20 +68,58 @@ export const EditorSensorItem: React.FC<Props> = (props: Props) => {
...
+          value={sensor.valueDisplay}
+          onChange={(event) => {
+            updateSensor((sensor) => {
+              sensor.valueDisplay = event.currentTarget.checked;
+            });
+          }}
+        />
+      </Field>
+
+      <Field label="IconName">
```
Comment (pierosavi) [inline src/customEditors/EditorSensorItem.tsx:104]:
```suggestion
      <Field label="Icon" description="Write a valid FontAwesome icon">
```

I noticed that some icons like `amazon` from here https://fontawesome.com/icons don't work, do you know why?
Do you think it's possible to retrieve a list of possible icons from the library and present it as a select?

File: src/module.tsx [hunk 2]
```diff
@@ -40,7 +40,7 @@ export const plugin = new PanelPlugin<SimpleOptions>(ImageItPanel).setPanelOptio
       description: 'List of sensors',
       category: ['Sensors'],
       defaultValue: [],
-      editor: props => {
+      editor: function SensorListEditor(props) {
```
Comment (pierosavi) [inline src/module.tsx:43]: Why changing the arrow function?

**Review by pierosavi — CHANGES_REQUESTED** [review]
Thanks for the PR! Overall it looks great to me, just some notes and then I can merge it and release a new version 🚀 👍🏻

File: src/module.tsx (hunk 2 above)
Comment (zfleet) [reply src/module.tsx:43]: That was suggested by the grafana toolkit

**Pull request merged**